src/Test.UnitTests.Sarif/GitHelperTests.cs

```diff
@@ -178,7 +178,7 @@ public void GitExePath_WhenPathDoesNotExist_SettingManuallyShouldWork()
         [Fact]
         public void GetTopLevel_WhenRepoPathIsToAFile()
         {
-            string pathToFile = typeof(GitHelperTests).Assembly.Location;
+            string pathToFile = Environment.CurrentDirectory;
```

Comment:
>Environment [](http://example.com/codeflow?start=32&length=11)

When tests execute under .net the product/test binaries are copied to a location outside of the repo.

Reply:
should the `pathToFile` point to a file?